src/commands/cmd_json.cc

```diff
@@ -52,7 +52,33 @@ class CommandJsonGet : public Commander {
   }
 };
 
+class CommandJsonStrAppend : public Commander {
+public:
+    Status Execute(Server *svr, Connection *conn, std::string *output) override {
+      redis::Json json(svr->storage, conn->GetNamespace());
+
+      JsonValue result;
+      std::vector<std::string> paths;
+      std::vector<std::string> values;
+      for (int i=2;i<(int)args_.size();++i) {
+        auto val = args_[i];
+        if (val.substr(0, 1) == "$") {
+          paths.push_back(val);
+        } else{
+          values.push_back(val);
+        }
+      }
+      uint64_t cnt = 0;
```

Comment:
The return value should be an array of int like https://github.com/apache/kvrocks/pull/1837 rather than a int